tools/remark-preset-lint-node/index.js

```diff
@@ -0,0 +1,44 @@
+// @see https://github.com/nodejs/node/blob/master/doc/STYLE_GUIDE.md
+
+'use strict';
+
+module.exports.plugins = [
+  require('remark-lint'),
+  require('remark-lint-checkbox-content-indent'),
+  require('remark-lint-definition-spacing'),
+  require('remark-lint-fenced-code-flag'),
+  require('remark-lint-final-definition'),
+  require('remark-lint-final-newline'),
+  require('remark-lint-hard-break-spaces'),
```

Comment:
It is a pity that `remark-lint-no-trailing-spaces` ["has not been updates for remark-lint 6.0.0"](https://github.com/wooorm/remark-lint#list-of-external-rules). Can we still use it?